pkg/store/sqlite/transactions.go

```diff
@@ -19,7 +19,7 @@ import (
 // Transaction CRUD interface
 //==========================================================================
 
-const listTransactionsSQL = "SELECT id, source, status, counterparty, counterparty_id, originator, originator_address, beneficiary, beneficiary_address, virtual_asset, amount, last_update, created, modified FROM transactions"
+const listTransactionsSQL = "SELECT t.*, count(e.id) AS numEnvelopes FROM transactions t LEFT JOIN secure_envelopes e ON t.id=e.envelope_id GROUP BY t.id ORDER BY t.created DESC"
```

Comment:
Turns out that the problem was when there were no secure envelopes the default JOIN returned no rows; this is fixed by adding a LEFT JOIN to ensure that transactions with no secure envelopes are returned on the listing.

Reply:
Nice inclusion of ORDER BY here.